frontend/src/app/testQueue/QueueItem.tsx

```diff
@@ -428,20 +443,9 @@ const QueueItem: any = ({
     if (appInsights) {
       trackUpdateAoEResponse({});
     }
-
-    const symptomOnset = answers.symptomOnset
```

Comment:
Does this change belong on this branch? I feel like it's from the refactor in #1820 that DRYs up this formatting into utils

Otherwise, LGTM ✅ but please double check this before merging

Reply:
Addressed in a39cfc3.